agent/stats/engine.go

```diff
@@ -793,3 +820,43 @@ func (engine *DockerStatsEngine) ContainerDockerStats(taskARN string, containerI
 
 	return containerStats, containerNetworkRateStats, nil
 }
+
+// getContainerStatusMessage reads container's status message file, updates dockerIdToStatusMessageSince mapping
+// and returns the file contents when
+// 	1. reading the status message of the given container for the first time.
+// 	2. unable read the file info for modified time.
+// 	3. status message file has been modified since the last time it was published to TACS.
+//	4. status message contents has not been sent to TACS after 30 mins time interval since last published.
+// getContainerStatusMessage returns empty string when
+//	1. error while reading status message file.
+// 	2. status message has not been modified within 30 mins since last published.
```

Comment:
nit: maybe add add "and status message has been sent to TACS within 30 mins"

Reply:
Updated the comments.